src/bot/constants.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Different transaction statuses.
+ */
+export const TRANSACTION_STATUS = {
```

Comment:
@tcoratger Maybe will be clearer to use enum here? Example: https://github.com/grindery-io/grindery-wallet/blob/main/src/types/State.ts#L94-L100

Reply:
@milukove Fully aligned, addressed here: https://github.com/grindery-io/grindery-bot-auth/pull/187